examples/index.jsx

```diff
@@ -0,0 +1,97 @@
+/* global cozy */
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Layout, Main } from 'cozy-ui/transpiled/react/Layout'
+import CozyClient, { CozyProvider, withClient } from 'cozy-client'
+import { Route, hashHistory } from 'react-router'
+import { MobileRouter } from 'cozy-authentication'
+import 'cozy-ui/transpiled/react/stylesheet.css'
+import appIcon from './icon.png'
+
+const minilog = window.minilog
+minilog.enable()
+minilog.suggest.allow('cozy-realtime', 'debug')
+
+// TODO ui: AppLinker should not fail so loudly without lampa.startapp plugin
+window.startApp = {
+  set: function() {
+    return this
+  },
+  check: () => Promise.resolve(false),
+  start: resolve => resolve()
+}
+
+const Index = withClient(({ client }) => {
+  return (
+    <Layout>
+      <Main>
+        <pre>{JSON.stringify(client.options, null, 2)}</pre>
+        You are logged in !
+      </Main>
+    </Layout>
+  )
+})
+
+const appInfo = {
+  name: 'Example bar',
+  slug: 'example-bar',
+  softwareID: 'io.cozy.example',
+  redirectURI: 'http://localhost:1234/auth',
+  universalLinkDomain: 'cozyexample://'
```

Comment:
Can we have something more realistic for ul domain ? Like https://cozyexample.com ? ;) 

Also you should have a protocol props with 'cozyexample://'

Reply:
Done, well spotted, I mixed up the two :)